src/main/java/useCases/Calculator/helper/CalculateLocalDateTime.java

```diff
@@ -0,0 +1,59 @@
+package useCases.Calculator.helper;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+public class CalculateLocalDateTime {
+    private final LocalDateTime today;
+    private final LocalDateTime otherday;
+
+    public CalculateLocalDateTime(LocalDateTime today, LocalDateTime otherday){
```

Comment:
The class name doesn't seem to match its description. I was thinking something more like 
CalculateTimeDifference as that seems to be what the class is responsible for.